packages/README.md

```diff
@@ -45,6 +45,14 @@ The new package should be automatically tested with `yarn test` in CI by the `te
 
 To test your package locally, you can run `yarn test_packages bar`. If you package depends on other packages you must run `yarn build_packages foo` to build a specific dependency or `yarn build_packages_all` to build all dependencies.
 
+### Providing a specific seed for deterministic testing
```

Comment:
this README is for developing in this repo
user documentation is published at https://www.npmjs.com/package/@bazel/jasmine
you can edit that documentation here: https://github.com/bazelbuild/rules_nodejs/blob/master/packages/jasmine/src/jasmine_node_test.bzl#L36